src/logistration/Logistration.jsx

```diff
@@ -39,6 +39,7 @@ const Logistration = (props) => {
   const [key, setKey] = useState('');
   const navigate = useNavigate();
   const disablePublicAccountCreation = getConfig().ALLOW_PUBLIC_ACCOUNT_CREATION === false;
+  const hideRegistrationLink = getConfig().SHOW_REGISTRATION_LINKS === false;
```

Comment:
Add `SHOW_REGISTRATION_LINKS` flag in src/config/index.js to utilize this flag using `getConfig`

Reply:
@mubbsharanwar  Done. I'd like to point out that because the default value of this param is `true`  and AFAIK there is no way to specify a boolean as an env variable, I resorted to using an explicit comparison with the string `'false'`.